bot/exts/evergreen/quack.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+
+class Quack(commands.Cog):
+    """Cog for the .quack command."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, seed: Optional[str], ducktype: Optional[str] = None) -> None:
+        """
+        Pulls a random URL from the Quackstack API.
+
+        If a seed is provided, a duck is generated based on the given seed.
+        If a duck type is provided, either a duck or manduck will be generated.
+        """
+        quackstack_url = "https://quackstack.pythondiscord.com/"
+
+        image_url = "https://quackstack.pythondiscord.com"
```

Comment:
We should define the API URL once, as a constant (all caps), at the start of the file.